archive.istio.io/404.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```

Comment:
this is already pretty much the 404 that the scripts put there (the one from the last build)